doc/howto/import_machines_to_instances.md

```diff
@@ -27,13 +27,21 @@ The tool can create both containers and virtual machines:
 
 * When creating a container, you must provide a disk or partition that contains the root file system for the container.
   For example, this could be the `/` root disk of the machine or container where you are running the tool.
-* When creating a virtual machine, you must provide a bootable disk, partition or image.
+* When creating a virtual machine, you must provide a bootable disk, partition, or an image in raw, QCOW, QCOW2, VDI, VHDX, or VMDK format.
   This means that just providing a file system is not sufficient, and you cannot create a virtual machine from a container that you are running.
   It is also not possible to create a virtual machine from the physical machine that you are using to do the migration, because the migration tool would be using the disk that it is copying.
   Instead, you could provide a bootable image, or a bootable partition or disk that is currently not in use.
 
+The tool can also inject the required drivers into the image:
+
+* Use `lxd-migrate --conversion=format,virtio` to ensure the image is converted into the raw format and virtio drivers are injected during the conversion.
+  The conversion option `virtio` requires `virt-v2v-in-place` to be installed on the host where LXD server runs.
```

Comment:
````suggestion
* To convert the image into raw format and inject the VIRTIO drivers during the conversion, use the following command:

       lxd-migrate --conversion=format,virtio

  ```{note}
  The conversion option `virtio` requires `virt-v2v-in-place` to be installed on the host where the LXD server runs.
  ```
````
To make it a bit easier to scan and find the important information.

Reply:
Thanks.